players.md

```diff
@@ -0,0 +1,152 @@
+| **TTML**                | ✅                                                                                                             | 🟡                                                                           |
+| **VTT**                 | ✅                                                                                                             | 🟡                                                                           |
+| **XSub**                | ✅                                                                                                             | ❌                                                                           |
+
+## Notes
+
+**Subtitles are not currently supported in the Native Player.** 
+
+---
+
+## HDR Support
+
+| Subtitle Format         | Swiftfin (VLCKit)                                                                                               | Native (AVKit)                                                               |
+|-------------------------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|
+| **Dolby Vision Profile 5**              | ❌                                                                                                             | ✅                                                                           |
+| **Dolby Vision Profile 8**              | ❌                                                                                                             | 🟡 Compatible devices only                                                                         |
+| **Dolby Vision Profile 10**              | ❌                                                                                                             | ✅                                                                           |
+| **HDR10**                 | ❌                                                                                                             | ✅                                                                           |
+| **HDR10+**              | ❌                                                                                                             | 🟡 Limited support                                                                           |
+| **HLG**                 | ❌                                                                                                             | ❌                                                                           |
+
+## Notes
+
+- HDR10+ support in Native (AVKit) is limited to certain devices, such as the Apple TV 4K (3rd Generation) and recent iPhones and iPads with compatible hardware.
+- HLG (Hybrid Log-Gamma) support in Native (AVKit) is limited and may vary by device and OS version.
+- Dolby Vision Profile 10 (used in AV1) is currently not supported by Native (AVKit).
```

Comment:
I'm not sure this is technically accurate. I haven't tested it, but it wouldn't surprise me if it worked in Native for the very limited amount of devices that support AV1 hardware decode (iPhone 15 Pro+)